docs/src/main/sphinx/release/release-464.md

```diff
@@ -0,0 +1,41 @@
+# Release 464 (30 Oct 2024)
+
+## General
+
+* {{breaking}} Require JDK 23 to run Trino. ({issue}`21316`)
+* Add the [](/connector/faker) to allow easy generation of data. ({issue}`23691`)
+* Add the Vertica connector. ({issue}`23948`)
+* Rename the
```

Comment:
Breaking for those who have used the old property, yeah?

Reply:
The old ones are still supported, but deprecated.